packages/dynamo-store-indexer-lambda/src/lib/Handler.ts

```diff
@@ -0,0 +1,70 @@
+import { DynamoDBStreamEvent } from "aws-lambda"
+import { AppendsEpoch, AppendsIndex, AppendsTrancheId, DynamoStoreIndexer, IndexStreamId } from "@equinox-js/dynamo-store-indexer"
+import { SpanStatusCode, trace } from "@opentelemetry/api"
+
+const tracer = trace.getTracer("@equinox-js/dynamo-store-indexer-lambda")
+
+const parse = (records: DynamoDBStreamEvent["Records"]): AppendsEpoch.Events.StreamSpan[] => {
+  const span = tracer.startSpan("parseEvent")
+  const spans: AppendsEpoch.Events.StreamSpan[] = []
+  let summary = "",
+    indexStream = 0,
+    noEvents = 0
+  try {
+    for (const record of records) {
+      if (record.dynamodb?.StreamViewType !== "NEW_IMAGE" && record.dynamodb?.StreamViewType !== "NEW_AND_OLD_IMAGES") {
+        throw new Error("Unexpected StreamViewType " + record.dynamodb?.StreamViewType)
+      }
+      summary += String(record.eventName)[0]
+      const updated = record.dynamodb.NewImage!
+      switch (record.eventName) {
+        case "REMOVE":
+          break
+        case "INSERT":
+        case "MODIFY": {
+          const p = record.dynamodb.Keys!.p.S!
+          const sn = IndexStreamId.ofString(p)
+          const n = Number(updated.n.N)
+          const appendedLen = Number(updated.a.N)
+          if (p.startsWith(AppendsEpoch.Category) || p.startsWith(AppendsIndex.Category)) indexStream++
```

Comment:
just check for starts with $ https://github.com/jet/propulsion/issues/197

Reply:
Thanks for the heads up 🙏